src/App.js

```diff
@@ -2,19 +2,31 @@ import React, { useEffect } from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
-function useSemiPersistentState() {
-  const [todoList, setTodoList] = React.useState(JSON.parse(localStorage.getItem("savedTodoList")) || []);
 
+function App() {
+ 
+  const [todoList, setTodoList] = React.useState([]);
+  const [isLoading, setIsLoading]=React.useState(true);
   useEffect(() => {
-    localStorage.setItem("savedTodoList", JSON.stringify(todoList));
+    new Promise((resolve, reject)=>{
+      setTimeout(() => {
+        resolve({data:{
+          todoList:JSON.parse(localStorage.getItem("savedTodoList")),
+        }})
+      }, 2000);
```

Comment:
Just a couple of minor spacing issues above:
- Line 9: `] = React.useState`
- Line 11: `reject) => {`
- Line 14: `todoList: JSON.parse(`

You do these correctly in other places so I know you know what is wrong, but I think it's worth pointing it out for you, as it is obvious to someone reading the code.